cypress/page/modal.page.ts

```diff
@@ -0,0 +1,34 @@
+class Modal{
+    private tableSelector: String;
+    constructor(){
+        this.tableSelector = ".modal-body tbody";
+    }
+    public checkingModal(personalInformation: any): void {
```

Comment:
as I mentioned before, you can declare an interface or type to represent the personalInformation object